src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -17,7 +17,6 @@
    [broadfcui.net :as net]
    [broadfcui.page.workspace.create :as create]
    [broadfcui.persistence :as persistence]
-   [broadfcui.utils :as utils]
```

Comment:
We keep this import, so you don't have to mess with imports to put a log on something.